invenio_deposit/static/js/invenio_deposit/app.js

```diff
@@ -24,12 +24,10 @@
 (function (angular) {
   // Bootstrap it!
   angular.element(document).ready(function() {
+
     angular.bootstrap(
       document.getElementById("invenio-records"), [
-        'invenioRecords', 'schemaForm', 'mgcrea.ngStrap',
-        'mgcrea.ngStrap.modal', 'pascalprecht.translate', 'ui.select',
-        'mgcrea.ngStrap.select', 'invenioFiles'
-      ]
+      'invenioRecords', 'schemaForm', 'mgcrea.ngStrap', 'mgcrea.ngStrap.modal', 'pascalprecht.translate', 'ui.sortable', 'ui.select', 'mgcrea.ngStrap.select', 'mgcrea.ngStrap.datepicker', 'mgcrea.ngStrap.helpers.dateParser', 'mgcrea.ngStrap.tooltip', 'invenioFiles'      ]
```

Comment:
I don't want to nitpick but why is this really long line better then the above?

Reply:
It is not! My editor was wrapping those lines. Fixed.